### PR TITLE
feat: add platform and CPU_ONLY parameters to enable smoke testing on all platform types

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [ ] Added label to the Pull Request for easier discoverability and search
- [ ] Commit Message meets guidelines as indicated in the URL https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md
- [ ] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

feat: add platform and CPU_ONLY parameters to enable smoke testing on all platform types

## Issue this PR will close

close: #**issue_number**

## Test Instructions if applicable

build base images
build benchmark images
run "./smoke_test.sh core 0

### CODE REVIEW

File: benchmark-scripts/smoke_test.sh [hunk 1]
```diff
@@ -8,26 +8,49 @@
...
 sudo rm -rf results || true
-sudo rm -rf smoke_test_camera_simulator_full/  || true
-sudo rm -rf smoke_test_camera_simulator_yolov5/  || true
-sudo rm -rf smoke_test_file_full/ || true
-sudo rm -rf smoke_test_stream_density/ || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_camera_simulator_full/  || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_camera_simulator_yolov5_classification/  || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_camera_simulator_yolov5/  || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_file_full/ || true
+sudo rm -rf "$PLATFORM"_"$CPU_ONLY"_smoke_test_stream_density/ || true
```
Comment (jim-wang-intel) [inline benchmark-scripts/smoke_test.sh:29]: suggest changing the file name with explicitly spelling out the platform and cpu only:
like :
```suggestion
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_camera_simulator_full/  || true
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_camera_simulator_yolov5_classification/  || true
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_camera_simulator_yolov5/  || true
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_file_full/ || true
sudo rm -rf "platform_$PLATFORM"_"cpuonly_$CPU_ONLY"_smoke_test_stream_density/ || true
```

Comment (brian-intel) [reply benchmark-scripts/smoke_test.sh:29]: I like it! will merge your suggestion

**Comment by jim-wang-intel** [comment]
oops, there is shellcheck warning to be fixed..

**Comment by brian-intel** [comment]
Saw those and will address them

**Review by vli11 — APPROVED** [review]
LGTM

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**